xtevent/xteventplot.ado

```diff
@@ -622,7 +622,7 @@ end
 * Program to parse cmdline and return commands for overlay static plot
 
 program define parsecmdline, rclass
-	syntax anything [aw fw pw] [if][in], samplevar(string) [Window(numlist min=1 max=2 integer) savek(string) plot proxy(string) POLicyvar(string) impute(string) *]
+	syntax anything [aw fw pw] [if][in], samplevar(string) [Window(string) savek(string) plot proxy(string) POLicyvar(string) impute(string) *]
 	

```

Comment:
@Constantino-Carreto-Romero Just confirming something. We don't need to include the window option parser inside xteventplot here because the window option is already "valid", in the sense that it was already parsed and passed checks when ran by xtevent. Is that so?

Reply:
@jorpppp 
Yes, if `xtevent` completed execution, `window(max | balanced)` passed all checks and is valid. When executing `xteventplot` and parsing `cmdline` for overlay plots, for instance for the overlay static plot `xteventplot` will call `xtevent` in https://github.com/JMSLab/xtevent/blob/267cef77ee4a32a1c26cce9d41e129d88cd45be0/xtevent/xteventplot.ado#L252 using the original specification of `window`: `window(max)` or `window(balanced)`. It will be passed to `xtevent` along with the `impute` option because it was specified and passed checks in the original call to `xtevent`. Therefore, there are not inconveniences with `xteventplot` passing to `xtevent` the original specification of `window`.
The following example covers that case and runs without inconveniences
```
use example31, clear
xtevent y eta , panelvar(i) timevar(t) policyvar(z) impute(stag) window(max)
xteventplot, overlay(static)
```